webapp-django/crashstats/profile/tests/test_views.py

```diff
@@ -0,0 +1,125 @@
+import pyquery
+
+from nose.tools import eq_, ok_
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+
+from crashstats.crashstats.management import PERMISSIONS
+from crashstats.supersearch.models import SuperSearchFields, SuperSearch
+from crashstats.crashstats.tests.test_views import BaseTestViews
+
+
+class TestViews(BaseTestViews):
+
+    def _login(self):
+        user = User.objects.create_user('test', 'test@mozilla.com', 'secret')
+        assert self.client.login(username='test', password='secret')
+        return user
+
+    def test_profile(self):
+
+        def mocked_supersearchfields(**params):
+            return {
+                'email': {
+                    'name': 'email',
+                    'query_type': 'string',
+                    'namespace': 'processed_crash',
+                    'form_field_choices': None,
+                    'permissions_needed': ['crashstats.view_pii'],
+                    'default_value': None,
+                    'is_exposed': True,
+                    'is_returned': True,
+                    'is_mandatory': False,
+                }
+            }
```

Comment:
You don't need to change this I think, the mocking done by default in the base class should be enough for you. Though maybe you are doing something specific here, but if not I would try to remove this to make the test smaller.